handbook/engineering/core-services/index.md

```diff
@@ -1,11 +1,66 @@
 # Code services team
 
-[Roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.fv5i7qi85bru)
-
 ## Vision statement
 
-Sourcegraph operations are fast (e.g. search), instances scale to 80k repositories, private code is secure and respects ACLs for every authorization provider, and the product settings are easy to set up and understand for our largest customers. Tech stack: Go, Postgres, Redis, Docker, Kubernetes, and others.
+Sourcegraph operations are fast (e.g. search), instances scale to 80k repositories, private code is secure and respects ACLs for every authorization provider, and the product settings are easy to set up and understand for our largest customers.
 
 ## Tech stack
 
-Go, Postgres, Redis, Docker, Kubernetes.
\ No newline at end of file
+Go, Postgres, Redis, Docker, Kubernetes.
+
+## Resources
+
+- [Roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.fv5i7qi85bru)
+- [Tracking Issues](https://github.com/sourcegraph/sourcegraph/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Ateam%2Fcore-services+label%3Atracking)
+
+## Processes
+
+### Weekly check-ins
+
+Everyone writes a weekly check-in at the end of each Friday in the active tracking issue of the on-going iteration. These check-ins describe the work done in the previous week as well as what's to come in the next week.
```

Comment:
Slightly nitpicky, but the last two times we did this Friday _and_ Monday. The "good weekly check-in" you link to below was also made on a Monday.

Reply:
Just an idea, could we say "(xx hours) before the core services team sync"? It's more closer to the sync with most recent updated status. Or should we post updates (edit comment/another comment) after Friday one if any?

Reply:
The Friday schedule is intentional, but up for discussion. My thought is that it provides for a great way to wrap-up the week and think about what you'll want to do in the next one, allowing your brain to shut-off from work for the weekend (tip from the Deep Work book).

Reply:
I personally prefer Monday for planning and reflecting on the previous week. In practice that is what people have been doing. I like @unknwon suggestion, maybe include what you suggest with motivation for why Friday is useful. But I'd like to stick with Mondays.